src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -101,10 +108,24 @@ export class TimeClockComponent  implements OnInit {
   }
 
   setTimeToInOut() {
-    this.currentDate = new Date();
-    this.hour = this.currentDate.getHours();
-    this.minute = this.currentDate.getMinutes();
-    this.seconds = this.currentDate.getSeconds();
+    if ( this.execOnlyOneTimeClockIn === 0 ) {
+      this.currentDate = new Date();
+      this.hour = this.currentDate.getHours();
+      this.minute = this.currentDate.getMinutes();
+      this.seconds = this.currentDate.getSeconds();
+      this.execOnlyOneTimeClockIn++;
+    }
+
+  }
+
+  setVarToEmpty() {
```

Comment:
can you find a better name like: `setDefaultValuesToFields`?

Reply:
done